third_party/istio-latest/istio-ci-mesh.yaml

```diff
@@ -43,8 +43,6 @@ spec:
   addonComponents:
     pilot:
       enabled: true
-    prometheus:
```

Comment:
out of curiosity, why does this need to be deleted?

Reply:
details here: https://github.com/knative/serving/issues/10221